R/get_join_keys.R

```diff
@@ -0,0 +1,20 @@
+#' Function to get join keys from a `` object
+#' @param data `` - object to extract the join keys
+#' @return Either `JoinKeys` object or `NULL` if no join keys
+#' @export
+get_join_keys <- function(data) {
```

Comment:
If you introduce a new method please check for possible replacement of old method for consistency reasons
```
❯ grep -r "get_join_keys()" --include="*.R" .
./R/module_teal_with_splash.R:        keys = data$get_join_keys()
./R/module_teal_with_splash.R:            keys = data$get_join_keys()
./R/module_nested_tabs.R:    datasets$get_join_keys(),
```